app/assets/stylesheets/style.css.scss

```diff
@@ -585,6 +585,14 @@ ul.gray-box.small > li + li {
   color: red;
 }
 
+/* Disable Bootstrap styling to allow for scrolling within annotations and
```

Comment:
Perhaps this style is better off with speedgrader related styling in `annotations.scss`?